path_roles.go

```diff
@@ -167,30 +186,43 @@ func (b *tfBackend) pathRolesWrite(ctx context.Context, req *logical.Request, d
 		return logical.ErrorResponse("ttl cannot be greater than max_ttl"), nil
 	}
 
-	if err := setTerraformRole(ctx, req.Storage, name, roleEntry); err != nil {
+	// if we're creating a role to manage a Team or Organization, we need to
+	// create the token now. User tokens will be created when credentials are
+	// read.
+	if roleEntry.Organization != "" || roleEntry.TeamID != "" {
```

Comment:
I wonder if it's worth returning a resp.Warning here indicating that this may have overwritten any token for the org/team that existed outside of Vault.

Reply:
Possibly? It's mentioned in both the usage and API docs in the documentation: 

- https://github.com/hashicorp/vault/pull/10823

I wonder about the effectiveness of issuing a warning at this point; it's only after the fact do we warn them in the CLI, we can't warn them ahead of time of this action. So it's less of a warning and more of a note?